src/main/resources/config/config.yaml

```diff
@@ -13,9 +13,10 @@ logging:
       threshold: ALL
       timeZone: UTC
```

Comment:
do we use this timeZone when creating the logger appender?

Reply:
we do but it seems like its UTC by default: https://github.com/dropwizard/dropwizard/blob/master/dropwizard-logging/src/main/java/io/dropwizard/logging/AbstractAppenderFactory.java#L116

Reply:
But is this only used by the layout builder, which we skip out entirely? We could use this value and set the timeZone on the Logstash encoder, but currently I think we ignore it.

Reply:
I don't think it related to the layout builder? Our LogstashConsoleAppenderFactory is a subclass of the AbstractAppenderFactory so the timeZone field is used directly.

Reply:
If you dig into the source for `AbstractAppenderFactory`, the timeZone only gets accessed by the `buildLayout` method which will never be called. We only use `AbstractAppenderFactory` for the `wrapAsync` method I think

Reply:
ok, i've removed this.